client/tinychain/ml/optimizer.py

```diff
@@ -13,29 +14,54 @@ class Optimizer(Map):
     def optimize(self, i, gradient, inputs, loss):
         """Update the given `gradient` by computing deltas for the given `loss`."""
 
+    @property
+    def lr(self) -> F32:
+        return self['lr']
+
 
 class GradientDescent(Optimizer):
     @classmethod
-    def create(cls, learning_rate=F32(0.01)):
+    def create(cls, lr=F32(0.01)):
```

Comment:
nit: I think `learning_rate` is a more descriptive name, but if you think `lr` makes more sense then you should update the docstring also